src/styles/blocks/container.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
@DarkMistyRoom if set it as:

<div class="catalog catalog--position">

and then: 

.catalog--position {
  margin: 0 auto;
}

will it be correct? sorry, this BEM is still kinda hard for me, I don't fully understand it.

Reply:
better use `container__????` because catalog placed inside container block